src/xenia/gpu/dxbc_shader_translator.cc

```diff
@@ -435,12 +435,19 @@ void DxbcShaderTranslator::StartVertexOrDomainShader() {
                                                   : dxbc::Dest::R(0, 0b0111),
                  dxbc::Src::VDomain(0b000110));
         if (register_count() >= 2) {
-          // Copy the primitive index to r1.x as a float.
+          // Copy the primitive index + offset to r1.x as a float.
           uint32_t primitive_id_temp =
               uses_register_dynamic_addressing ? PushSystemTemp() : 1;
           in_primitive_id_used_ = true;
+          a_.OpMov(dxbc::Dest::R(primitive_id_temp, 0b0001),
```

Comment:
Shader inputs can be used directly as source operands, there's no need to `mov` it.